pkg/ebpf/tracer_windows.go

```diff
@@ -47,23 +62,55 @@ func init() {
 type Tracer struct {
 	config       *Config
 	driverHandle windows.Handle
+	iocp         windows.Handle
+	bufs         []readBuffer
+	packetCount  int64
 }
 
 // NewTracer returns an initialized tracer struct
 func NewTracer(config *Config) (*Tracer, error) {
 	handle, err := openDriverFile(driverFile)
 	if err != nil {
-		return nil, fmt.Errorf("%s : %s", "Could not create driver handle", err)
+		return nil, fmt.Errorf("%s : %s", "could not create driver handle", err)
+	}
+
+	// Create IO Completion port that we'll use to communicate with the driver
+	iocp, err := windows.CreateIoCompletionPort(handle, windows.Handle(0), 0, 0)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create IO completion port %v", err)
 	}
 
 	tr := &Tracer{
 		driverHandle: handle,
+		iocp:         iocp,
+		bufs:         make([]readBuffer, totalReadBuffers),
+	}
+
+	// Set the packet filters that will determine what we pull from the driver
+	// TODO: Determine failure condition for not setting filter
+	// TODO: I.e., one or more, all fail? I.E., at what point do we not create a tracer
+	err = tr.prepareDriverFilters()
+	if err != nil {
+		return nil, fmt.Errorf("failed to setup packet filters on the driver: %v", err)
+	}
+
+	// Prepare the read buffers that will be used to send packets from kernel to user space
+	err = tr.prepareReadBuffers()
+	if err != nil {
+		return nil, fmt.Errorf("failed to prepare ReadBuffers: %v", err)
 	}
 
+	err = tr.initPacketPolling()
+	if err != nil {
+		log.Warnf("issue polling packets from driver")
+	}
 	go tr.expvarStats()
 	return tr, nil
 }
 
+// Stop function stops running tracer
+func (t *Tracer) Stop() {}
+
 func (t *Tracer) expvarStats() {
 	ticker := time.NewTicker(5 * time.Second)
```

Comment:
We should probably have `defer ticker.Stop()` here to clean it up when this exits.

Reply:
Agreed, we should also use whatever mechanism initPacketPolling uses for stopping to stop the stats from being reported